packages/react/src/runtimes/edge/converters/toLanguageModelMessages.ts

```
@@ -154,6 +154,9 @@ export function toLanguageModelMessages(
             case "source": {
               break; // reasoning and source parts are omitted
             }
+            case "source": {
```

Biome (1.9.4)

[error] Duplicate case label.

The first similar label is here:


(lint/suspicious/noDuplicateCase)